domains/glutis.json

```diff
@@ -4,6 +4,10 @@
     "email": "huyv80313@gmail.com"
   },
   "record": {
-    "CNAME": "huyvu15.github.io"
-  }
+    "CNAME": "huyvu15.github.io",
+    "TXT": "6f9160dd04b01ff24ed06141d1b683"
```

Comment:
Cannot use CNAME in conjunction with other records and vice versa. See docs for more information: https://docs.is-a.dev/

Make sure to also include a GitHub verification record in another file. You can find out more information on this at the docs: https://docs.is-a.dev/guides/github-pages